docs/core-concepts/libraries/libraries-with-cli.md

```diff
@@ -146,42 +137,32 @@ Here is an example targeting PCL Profile 328. Profile 328 supports: .NET Standar
 
 You can now build.
 
-Using DNX:
-
-```
-$ dnu restore
-$ dnu build
-```
-
-Using .NET CLI:
-
 ```
 $ dotnet restore
-$ dotnet compile
+$ dotnet build
 ```
 
-You will now notice two new entries in your `/bin/Debug` folder:
+You will now notice the following entry in the `/bin/Debug` folder:
 
 ```
-/bin
-   /Debug
-      /dotnet55
-      /portable-net40+sl50+netcore45+wpa81+wp8
+$ ls bin/Debug
+
+portable-net40+sl50+netcore45+wpa81+wp8/
 ```
 
-Each folder contains distinct `.dll` files corresponding to their named target.
+This folder contains the `.dll` files necessary to run your library.
 
-## How to cross-compile for .NET Core and .NET Framework
+## How to Multitarget
 
-**NOTE:** These instructions assume you have .NET Framework installed on your machine.  Refer to the [Prerequisites](#prerequisites) to get dependencies installed.
+**NOTE:** These instructions assume you have the .NET Framework installed on your machine.  Refer to the [Prerequisites](#prerequisites) to get dependencies installed.
 
-You may need to support older versions of .NET Framework when supporting both .NET Framework and .NET Core.  You may also wish to use newer APIs and language constructs for newer targets.  You can do this by using `#if` directives.
+You may need to support older versions of the .NET Framework when supporting both the .NET Framework and .NET Core.  You may also wish to use newer APIs and language constructs for newer targets.  You can do this by using `#if` directives.
```

Comment:
By reading this paragraph again, the concepts seem a little lose and it's a bit confusing.

My suggestion (please make sure I haven't introduced any inaccuracies):
You may need to target older versions of the .NET Framework when your project supports both the .NET Framework and .NET Core. In this scenario, if you want to use newer APIs and language constructs for the newer targets, use `#if` directives in your code. You also might need to add different packages and dependencies in your project.json file for each version/platform you're targeting to include the different APIs needed for each case.
For example, let's say you have an app that performs networking operations over HTTP. For .NET Core and the .NET Framework versions 4.5 or higher, you can use the `HttpClient` class from the `System.Net.Http` namespace. However, earlier versions of the .NET Framework don't have the `HttpClient` class, so you could use the `WebClient` class from the `System.Net` namespace for those instead. 
So, the `project.json` file would look similar to this: